### ISSUE TITLE
How do I set up the canvas instance?

### ISSUE THREAD

**Issue by doldsimo** [doldsimo/lti-1.3-canvas-lms#4]
_Originally posted by @Liziqiuqiu in https://github.com/doldsimo/lti-1.3-canvas-lms/issues/3#issuecomment-1861446609_

Hi Simon,
This is not an issue, but rather a question I've come across. However, I'm unsure where to seek help. I will remove this if it's not the appropriate place to post this question.

In the readme document, the second step is to set up a Canvas instance. I've followed the [Production Start](https://github.com/instructure/canvas-lms/wiki/Production-Start) tutorial and can access the instance. However, I'm unable to locate the login page; I don't know how to login into the system; all I can find is the public directory shown below. Could you please provide some guidance?
![image](https://github.com/doldsimo/lti-1.3-canvas-lms/assets/132622020/55256f13-53ca-4bd0-a915-e3fc6d901882)

Thank you so much!!!

Have a wonderful Christmas season ahead!

_Originally posted by @Liziqiuqiu in https://github.com/doldsimo/lti-1.3-canvas-lms/issues/3#issuecomment-1861446609_

**Comment by doldsimo** [comment]
Hi  @Liziqiuqiu,

Yes, you are right, this is not a specific error or issue from this repository, but I would still like to try to help you.

It looks like you have Canvas set up, but the instance is probably not configured correctly.

Does your root directory contain an `index.html` or a `.conf`? Because the index.html is always considered as the entry point and in the .conf you can redefine the entry point.

I see that the website is not yet accessible via https and does not yet contain a domain. The Production Start also shows how to set up SSL, so I'm wondering if you completed the Production Start to the end.

If you need help, you can use [this video](https://www.youtube.com/watch?v=EHUP-1Arqhw&t=1121s&ab_channel=SoumaMaida), it shows the process for setting up the instance one-on-one. At the end it will also show you how to get to the canvas login page. I think this is exactly what you are looking for. I watched the video and he takes exactly the right steps to set up Canvas.

Hopefully this helps you, let me know if you need any further help.

Thank you, I wish you a nice Christmas season too.

**Comment by Liziqiuqiu** [comment]
Good morning Simon,
The information and video you provided are super helpful. I was struggling with the installation last weekend. I am not quite familiar with installing the Ruby application, so I missed some key configurations. I realized that the passenger does not run in my instance, which causes the failure to redirects to the login page. Also, I found several other people encountering the [same problem](https://github.com/instructure/canvas-lms/issues/2249), though I failed to address it after several hours of attempts. But at least, I installed the SSL certificates and am now able to access it with the [domain name](https://production.lmscanvas.online) (I am very excited). Next, I will compare the steps I have taken with the videos that you shared. I hope I can address it and have my first self-hosted Canvas up and running.

Again, thank you for your help and for being so patient. It is greatly appreciated.

Best wishes,
Qiuqiu
